sktime/forecasting/compose/_bagging.py

```diff
@@ -0,0 +1,304 @@
+#!/usr/bin/env python3 -u
+# -*- coding: utf-8 -*-
+# copyright: sktime developers, BSD-3-Clause License (see LICENSE file).
+"""Implements Bagging Forecaster."""
+
+__author__ = ["ltsaprounis"]
+
+from copy import deepcopy
+from typing import List, Union
+
+import numpy as np
+import pandas as pd
+from sklearn import clone
+from sklearn.utils import check_random_state
+from sklearn.utils._testing import set_random_state
+
+from sktime.datatypes._utilities import update_data
+from sktime.forecasting.base import BaseForecaster
+from sktime.transformations.base import BaseTransformer
+from sktime.transformations.bootstrap import STLBootstrapTransformer
+from sktime.utils.estimators import MockForecaster
+
+
+class BaggingForecaster(BaseForecaster):
+    """Bagged "Bootstrap Aggregating" Forecasts.
+
+    Bagged Forecasts are obtained by forecasting bootstrapped time series and then
```

Comment:
I think this section should give the reader enough information to tell what the algorithm is, precisely.

E.g., what exactly is bootstrapped, how? Is there a paper reference? If yes, which algorithm is it, from the paper reference?